modules/javafx.controls/src/main/java/javafx/scene/control/ListCell.java

```diff
@@ -352,6 +352,8 @@ public void changed(
      * Editing API                                                             *
      *                                                                         *
      **************************************************************************/
+    // index at time of startEdit - fix for JDK-8165214
+    int indexAtStartEdit;
```

Comment:
It seems like this field should be private.

Reply:
indeed, thanks :)